Gopkg.toml

```diff
@@ -0,0 +1,94 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "github.com/emicklei/proto"
+  version = "1.4.6"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/fullstorydev/grpcurl"
+
+[[constraint]]
+  name = "github.com/gogo/protobuf"
+  version = "0.5.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/golang/protobuf"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/grpc-ecosystem/grpc-gateway"
+
+[[constraint]]
+  branch = "master"
```

Comment:
Why are some of these branch, and some version?

Reply:
I’d like all dependencies to not have any version lock for now, same as glide was.

Reply:
Is it possible to have no version/branch locking in Dep?

Reply:
If you remove the constraint altogether it will automatically select the most recently tagged release. If you want to use `master` branch you have to create a constraint that specifies that.

Reply:
Yea I super appreciate the work here, thanks so much. Will play with it a bit.